test/topology_experimental_raft/test_alternator.py

```diff
@@ -140,7 +140,7 @@ async def get_cpu_metrics():
         for ip in ips:
             metrics = await manager.metrics.query(ip)
             ms_streaming += metrics.get('scylla_scheduler_runtime_ms', {'group': 'streaming'})
-            ms_statement += metrics.get('scylla_scheduler_runtime_ms', {'group': 'statement'})
+            ms_statement += metrics.get('scylla_scheduler_runtime_ms', {'group': 'sl:default'})
```

Comment:
The service level controller enforces a naming convention on the scheduling groups associated with service levels: `sl:<service level name>`. The "statement" scheduling group is reused as the scheduling group for the "default" service level, hence it is renamed to `sl:default`.

Moreover, the "default" service level is documented (https://enterprise.docs.scylladb.com/stable/using-scylla/workload-prioritization.html#using-workload-prioritization-with-your-application) as the service level used when no user-created service level applies (i.e. anonymous user or user without any SL assigned). Given how this is explained in the documentation, I think make sense for this scheduling group to now be named `sl:default`.

Reply:
I hope we don't have users, or a bazillion dtests, who rely on that old name "statement". I guess it won't be a problem because we've had this name "sl:default" in the Enterprise branch, and I don't remember anyone complaining.